### DISCUSSION TITLE
Cockroach DB Support?

### DISCUSSION THREAD

**Discussion by WoodyWoodsta** [outline/outline#6497] [Self Hosting] [answered]
Hey all. Been looking for a self-hosted KB for a while - Outline looks great.

Has anyone self-hosted Outline using CockroachDB? Wondering if there is anything that doesn't work before plugging it in and using it in our team.

**Comment by tommoor** [comment]
I don't know if anyone has tried it however it looks like it would be compatible based on the list of PostgresDB features missing in Cockroach.

**Reply by WoodyWoodsta** [reply]
Thanks @tommoor! Will report back once we've given it a whizz.

**Comment by WoodyWoodsta** [comment, marked as answer]
Unfortunately, the use of triggers at least [here](https://github.com/outline/outline/blob/main/server/migrations/20160711071958-search-index.js#L16) mean that CockroachDB will not work until https://github.com/cockroachdb/cockroach/issues/28296 is released.
